kuma/wiki/content.py

```diff
@@ -141,7 +141,10 @@ def code_sample(self, name):
             sample = pq('<section>%s</section>' % section)
         else:
             # If no section, fall back to plain old ID lookup
-            sample = pq(src).find('[id=%s]' % quoteattr(name))
+            try:
+                sample = pq(src).find('[id=%s]' % quoteattr(name))
+            except Exception:
```

Comment:
I'd like to restrict this to ``ValueError``, so that if a different exception is raised we'll still get the traceback.